hwdb/60-evdev.hwdb

```diff
@@ -76,9 +76,6 @@ evdev:input:b0003v05ACp0254*
  EVDEV_ABS_36=::92
 
 # MacbookPro10,1 (unibody, June 2012)
-evdev:input:b0003v05ACp0259*
-evdev:input:b0003v05ACp025A*
-evdev:input:b0003v05ACp025B*
```

Comment:
these should be fixed (range is 0262, 0263 and 0264), it's a simple copy-paste error. I sent a PR to fix these here: https://github.com/systemd/systemd/pull/3921